src/js/components/Ballot/PositionRowListCompressed.jsx

```diff
@@ -333,31 +342,56 @@ const CandidateEndorsementsContainer = styled('div')`
   align-items: flex-start;
   display: flex;
   flex-flow: column;
+  width: 100%;
```

Comment:
Please put these attributes in alphabetical order, as well as all of the other attributes you added.